### DISCUSSION TITLE
Welcome to cvlib Discussions!

### DISCUSSION THREAD

**Discussion by arunponnusamy** [arunponnusamy/cvlib#41] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wDove1** [comment]
hello, I have just added a pull request which I believe adds support for yolo v4 as I am using this library in a project for school

**Reply by arunponnusamy** [reply]
Thanks @wDove1. Will look into it.
